docs/contributing/testing.md

```diff
@@ -19,7 +19,17 @@ In your project folder, run `npm test`.
 
 Fix any CSS or JavaScript style errors that the linting tests report.
 
-## 2. Check your changes in the ‘review app’
+## 2. Fix any invalid HTML
+
+To check whether your HTML markup is valid, [check your page with W3C's markup validator](https://validator.w3.org/).
+
+If you can, fix any errors or warnings the validator reports.
+
+You should not need to fix any [known issues with our components](https://github.com/alphagov/govuk-frontend/issues/1280#issuecomment-509588851) that the validator reports.
+
+Let us know in your pull request, or [contact us](https://design-system.service.gov.uk/#support), if you need help with fixing an error or a warning.
```

Comment:
```suggestion
If you need help with fixing an error or a warning, let us know in your pull request or [contact us](https://design-system.service.gov.uk/#support).
```
I'd consider changing the order of this sentence, as I think the important bit is 'if you need help'. People who don't need help can stop reading there :) 

Everything else looks great 👍

Reply:
@sabrinaharris Great idea! Front-loads the important bit.